lib/ansible/plugins/loader.py

```diff
@@ -831,3 +831,11 @@ def _load_plugin_filter():
     C.DEFAULT_BECOME_PLUGIN_PATH,
     'become_plugins'
 )
+
+process_loader = PluginLoader(
+    'ProcessModel',
+    'ansible.plugins.process',
+    C.DEFAULT_PROCESS_MODEL,
```

Comment:
You need a new config for `DEFAULT_PROCESS_PLUGIN_PATH`, and reference that here.  Instead of just inserting the plugin name here.